### ISSUE TITLE
Vivid dreams

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6786]
**Preferred term label:**
Vivid dreams

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
https://pubmed.ncbi.nlm.nih.gov/23622114/

**Comment by pnrobinson** [comment]
looking at the article, I am not convinced this is a phenotypic feature that we can easily delineate. I am closing this for now, we can reconsider in the future if needed.
